app/controllers/custom_pages_controller.rb

```diff
@@ -13,11 +13,13 @@ def destroy
   private
 
   def upsert_custom_page
-    CustomPage.create!(params.permit(:translation_id, :page_id, :structure))
+    attrs = params[:data][:attributes]
+
+    CustomPage.create!(attrs.permit(:translation_id, :page_id, :structure))
```

Comment:
You also can use the AMS deserializer to pull off the correct attributes, if you want.
https://github.com/CruGlobal/missionhub-api/blob/master/app/controllers/apis/v4/base_controller.rb#L202